Solutions/VMWareESXi/Package/createUiDefinition.json

```diff
@@ -315,7 +315,7 @@
             "name": "huntingqueries-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs the following hunting queries. After installing the solution, run these hunting queries to hunt for threats in Manage solution view."
+              "text": "This solution installs the following hunting queries. After installing the solution, run these hunting queries to hunt for threats in Manage solution view. "
```

Comment:
Hi @v-rusraut Please discard this change as it is adding additional whitespace, thanks!

Reply:
Hi @v-vdixit 
Removed whitespace.
Thanks.